docs/use-linea/explore/use-vitidiary.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Use VitiDiary for Linea Voyage Quest
+description: Use the Vitidiary to experience the Learn to Earn game on Linea
+sidebar_position: 1
+---
+
+[VitiDiary](https://vitidiary.com) is A revolutionary wine-themed gaming experience. Enhance your wine knowledge, engage in quizzes, and create unique wines using NFTs. Unveil the world of wine!
+
+What is Viti Diary
+Viti Diary is a web3 game that combines blockchain technology, wine knowledge, and winemaking to create a unique gaming and learning experience.
+
+In Viti Diary, players can create their own unique wine NFTs by combining NFTs that represent grape varieties, terroirs, tools, and characters.
+
+Viti Diary is more than just a game - it's a learning platform that aims to educate players on viticulture, winemaking, and wine trading. With over 1,000 wine knowledge questions and over 500 winemaking recipes based on real-world techniques, players can develop their wine expertise while having fun.
+
+
+This step-by-step guide teaches how to complete the Game Tutorials and mint Voyage NFT via [Vitidiary](https://vitidiary.com/) tutorial and participate in Linea vs VitiDiary” quest.
+
+**Chapter 0. Preparation.**
+
+**Step 1.** **Download [MetaMask](https://metamask.io/download/)**
+
+**Step 2.** **Add Linea Goerli test network to your MetaMask.** Open your MetaMask, go to “Networks” -> “Add network” -> Put “Linea Goerli test network” in search and add it.
+
+![image](https://github.com/ConsenSys/doc.zk-evm/assets/45225985/0002e3b8-59e1-4655-9cc0-138de7236e98)
```

Comment:
can you do relative references to the image assets?

These links are broken

Reply:
<img width="855" alt="Screenshot 2023-06-13 at 10 51 19 AM" src="https://github.com/ConsenSys/doc.zk-evm/assets/17280653/f84375dc-964b-4019-a910-f12f86419664">

Reply:
fixed